### DISCUSSION TITLE
Frigate+ camera names, consequential for model generation or just for organization?

### DISCUSSION THREAD

**Discussion by nickcmeyer** [blakeblackshear/frigate#15593] [Ask A Question] [answered]
I have 2 instances of Frigate, mine and another set up for my parents property. I started sending images to Frigate+ and labeling before I thought about the fact that the camera "SideYard" exists in both system. This has lead to multiple camera views being under the same camera name in Frigate+. Is this going to have a negative effect on the model (I noticed camera settings has height for instance)? Should I delete photos associated with one of the cameras and change it's name before generating a model or is this something that doesn't really matter?

I'd like to correct this but not sure of the best way or time.

**Comment by NickM-27** [comment]
Currently I believe it doesn't matter

**Comment by blakeblackshear** [comment, marked as answer]
I would suggest fixing it going forward. I don't think it is worth going back and adjusting existing images.
